app/models/concerns/ats/greenhouse/application_fields.rb

```diff
@@ -1,8 +1,9 @@
 module Ats
   module Greenhouse
     module ApplicationFields
-      def get_application_question_set(job, data)
-        Importer::GetGreenhouseFields.call(job, data)
+      def get_application_question_set(_job, data)
```

Comment:
I'm wondering if we need to go to the modules anymore - presumably we can just call the service object directly if we know the ATS? One to discuss as the advantage of the current setup is that we already know which ATS this calls so wouldn't need to route this otherwise - it may not be an improvement to change the setup.